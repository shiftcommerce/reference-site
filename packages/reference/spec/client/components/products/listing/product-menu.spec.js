// Component
import ProductMenu from '../../../../../client/components/products/listing/product-menu'

test('renders correctly', () => {
  // Arrange
  const initialProps = {
    title: 'Super Products',
    body: 'Placeholder text can be changed using props'
  }

  // Act
  const wrapper = shallow(
    <ProductMenu {...initialProps} />
  )

  // Assert
  expect(wrapper).toMatchSnapshot()
  expect(wrapper.find('.c-product-listing__menu-description-title')).toIncludeText(initialProps.title)
  expect(wrapper.find('.c-product-listing__menu-description-content')).toIncludeText(initialProps.body)
})
