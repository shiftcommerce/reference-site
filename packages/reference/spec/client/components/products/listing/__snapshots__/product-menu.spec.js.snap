// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="c-product-listing__menu-description"
>
  <h1
    className="c-product-listing__menu-description-title"
  >
    Super Products
  </h1>
  <div
    className="c-product-listing__menu-description-content"
  >
    <p>
      Placeholder text can be changed using props
    </p>
  </div>
</div>
`;
